### DISCUSSION TITLE
Dashboard for all projects of an organization

### DISCUSSION THREAD

**Discussion by aiakubovich** [langfuse/langfuse#1258] [Ideas]
### Describe the feature or potential improvement

Hi, as the number of projects increases, it seems necessary to have a single overview page with aggregated results from all projects. Currently, if I add 10 projects, I need to check each one individually to review metrics such as token usage. It would be great if there were an option to have one dashboard with aggregated results. This dashboard could be configurable, allowing the user to select either all projects or only specific ones to display in the aggregated results. Thank you.

**Comment by marcklingen** [comment]
This is great, can be
- short-term: all projects a user has access to
- mid-term: org/team-level dashboard

Open for more suggestions on this and curious how many teams want this.

**Comment by udit-001** [comment]
I would really appreciate it if we could incorporate this feature into Langfuse. The current method of managing projects and providing access feels a bit counterintuitive. It seems like each user operates within their own isolated workspace, whereas I envision Langfuse as a tool that various teams can use to monitor multiple projects simultaneously.

As OP mentioned earlier, having a single dashboard for an overall view of all projects would be fantastic. To illustrate this idea, I'd like to reference Sentry and provide a screenshot for clarification.

![project viw](https://github.com/langfuse/langfuse/assets/23015406/dff2cb38-787d-4da0-97e9-28204bb88c35)

In the context of Langfuse, I imagine each card displaying the cost per project, allowing users to easily dive into specific projects by simply clicking on the respective card. Additionally, it might be beneficial to feature the total cost incurred across all projects prominently at the top.

**Reply by santhosh-motorq** [reply]
This would be great to have

**Reply by B-Mounir** [reply]
Definitely a mandatory feature !

**Reply by AkioNuernberger** [reply]
+1 request from a user!

**Reply by raviraval79** [reply]
Yes very much needed in enterprise platform setup with multiple tenant teams!

**Comment by irenebosque** [comment]
Any news about this request? is it going to be implemented in the future?

**Comment by alessandro-redi** [comment]
Hello @marcklingen, are there any update about this topic?

**Comment by vladbuinceanu** [comment]
Up

**Comment by T-Zaid** [comment]
Hello Maintainers / @marcklingen !

I'm interested in contributing to this feature.

I went through the original request and the current custom dashboard capabilities. From what I have seen in the application, dashboards are currently project-scoped, while this request in the discussion (having 30 upvotes and opened since 2024) asks to provide aggregated metrics across multiple projects.

I'm considering starting with a smaller MVP:

- An overview of projects the current user has access to.
- A selectable project filter.
- Aggregated LLM costs for a selected time range.
- A per-project cost breakdown with links to the existing dashboards.

Before diving into implementation or even proposing a solution, I'd appreciate your comments on these:

1. Is this feature still on the roadmap?
2. Would a user-accessible-projects approach be a reasonable first iteration?
3. Are there architectural or authorization constraints I should consider before proposing a design?

I'd be happy to investigate the existing dashboard and metrics architecture, put together a short technical proposal, and contribute the implementation if this aligns with the team's direction.

Thanks!
